src/components/tsconfig.json

```diff
@@ -10,5 +10,5 @@
     "noEmit": true,
     "esModuleInterop": true
   },
-  "files": ["index.d.ts", "index.tests.tsx"]
+  "files": ["index.d.ts", "index.test.tsx"]
```

Comment:
React Styleguidist will ignore .test but not .tests:

https://github.com/styleguidist/react-styleguidist/blob/master/docs/Configuration.md#ignore